ecommerce-product-page-main/designpage.html

```diff
@@ -1,161 +1,64 @@
-			Featuring a durable rubber outer sole,they'll withstand everthing the weather can offer.</p>
-			</div>
-
-			<div class="money">
-
-			<div class="containflex">
-
-				<div><h1 class="hfont">$125.00</h1></div>
-
-				<div><button class="button1">50%</button></div>
-
-			</div>
-
-					<div class="cash"><s>$250.00</s></div>
-
+	<head>
+		<meta name="viewpoint" content="width=device=width,initial-scale=1.0">
+		<title>design desktop/mobile responsive</title>
+		<link rel="stylesheet" href="./styled.css">
+	</head>
+	<body>
+		<div class="containpage">
+			<nav class="container">
+				<div class="list">
+					<img class="space" src="images/logo.svg">
+					<a href="#">Collection</a>
+					<a href="#">Men</a>
+					<a href="#">Women</a>
+					<a href="#">About</a>
+					<a href="#">Contact</a>
+				</div>
+				<div>
+					<img  class="cart" type="icon"  src="images/icon-cart.svg">
+					<img class="avatar"  type="icon"  src="images/image-avatar.png">
+				</div>
+			</nav>
+
+			<section class="containblock">
+				<div class="block1">
+					<img  class="img" src="images/image-product-1.jpg"/>
+					<div class="containthumbs">
+						<img src="images/image-product-1-thumbnail.jpg"> 
+						<img src="images/image-product-2-thumbnail.jpg">
+						<img src="images/image-product-3-thumbnail.jpg">
+						<img src="images/image-product-4-thumbnail.jpg">
 					</div>
-
-	<div class="containflexA">
-
-		<div class="containA">
-			<div class="buttonA"><button class="buttonB" >-</button></div>
-			<div class="number">0</div>
-			<div><button class="buttonC">+</button></div>
-		
+				</div>
+				<div class="block2">
+					<h5 class="h5orange">SNEAKER COMPANY</h5>
+					<h1 class="title">Fall Limited Edition Sneakers</h1>
+					<p class="description">These low-profile sneakers are your perfect causal wear companion. 
+						Featuring a durable rubber outer sole,they'll withstand everthing the weather can offer
+					</p>
+					
+					<div class="money">
+						<div class="containflex">
+							<h1 class="hfont">$125.00</h1>
+							<div><button class="discount">50%</button></div>
 						</div>
-
-					<div class="button20">
-						<button type="button" class="button2"><div>
-					<img  class="imgbutt"  src="images/icon-cart.svg" ></div>
-							<div ><h3 class="h3butt" >Add To Cart</h3></div></button>
+						<div class="cash"><s>$250.00</s></div>
 					</div>
-
-					
-	</div>
-
-	</div>
-
-
-
-
-
-
-
-
-
-
-
-
-</div>
-
-
-
-</div>
-</body>
-
+					<div class="counterSection">
+						<div class="addCounter">
+							<button class="counters " >-</button>
+							<div class="number">0</div>
+							<button class="counters">+</button>
+						</div>
+						<button type="button" class="addToCartButton">
+							<img  class="imgbutt"  src="images/icon-cart.svg" >
+							<h3 class="h3butt" >Add To Cart</h3>
+						</button>
+					</div>
+				</div>
+			</section>
+		</div>
+	</body>
 </html>
```

Comment:
Why was there no "dropbox" tag in your html file?

Reply:
i did not. must hae missed it